packages/server/src/conversations/api.ts

```diff
@@ -1,83 +1,54 @@
-import { Router } from 'express'
-import { Auth } from '../base/auth/auth'
+import { uuid } from '@botpress/messaging-base'
+import { Response } from 'express'
+import { ApiManager } from '../base/api-manager'
+import { ClientApiRequest } from '../base/auth/client'
 import { Schema } from './schema'
 import { ConversationService } from './service'
 
 export class ConversationApi {
-  constructor(private router: Router, private auth: Auth, private conversations: ConversationService) {}
+  constructor(private conversations: ConversationService) {}
 
-  async setup() {
-    this.router.post(
-      '/conversations',
-      this.auth.client.auth(async (req, res) => {
-        const { error } = Schema.Api.Create.validate(req.body)
-        if (error) {
-          return res.status(400).send(error.message)
-        }
-
-        const { userId } = req.body
-        const conversation = await this.conversations.create(req.client!.id, userId)
-
-        res.send(conversation)
-      })
-    )
+  setup(router: ApiManager) {
+    router.post('/conversations', Schema.Api.Create, this.create.bind(this))
+    router.get('/conversations/:id', Schema.Api.Get, this.get.bind(this))
+    router.get('/conversations/user/:userId', Schema.Api.List, this.list.bind(this))
```

Comment:
Should be:
```suggestion
    router.get('/conversations/users/:userId', Schema.Api.List, this.list.bind(this))
```

Reply:
I don't think so. We're listing `conversations` by their `user` property. I don't see the need for plural here.

Reply:
Usually entities in REST routes are all plural no matter what.